terraform-modules/cur-setup-destination/main.tf

```diff
@@ -195,7 +195,7 @@ resource "aws_cur_report_definition" "this" {
   time_unit                  = "HOURLY"
   format                     = "Parquet"
   compression                = "Parquet"
-  additional_schema_elements = ["RESOURCES"]
```

Comment:
@sean-nixon can you check if this could be optional? 

SCAD can bring lots of additional data that will impact the volume of CUR for big customers
Should it be rather optional?

Reply:
Yes, this could and should absolutely be optional. This can be controlled by a new input variables in `variables.tf`. I would suggest the following, defaulting to false as with CFN:

```
# variables.tf
...
variable "enable_split_cost_allocation_data" {
  description = "Enable split cost allocation data for ECS and EKS for this CUR report"
  default     = false
}
```

```
# main.tf
...
additional_schema_elements = var.enable_split_cost_allocation_data ? ["RESOURCES", "SPLIT_COST_ALLOCATION_DATA"] : ["RESOURCES"]
...
```

A couple other suggestions:
* This needs to be updated in both `cur-setup-source` and `cur-setup-destination` modules to the various deployment configurations possible.
* Lastly, since a variable is being added, the README files should be updated. These are auto-generated using [terraform-docs](https://terraform-docs.io/) by running `terraform-docs .` from each module directory.

Reply:
@MaximF can you check this thread?

Reply:
@iakov-aws  and @sean-nixon  I just pushed the changes, please review it again.